src/lib/content.ts

```diff
@@ -32,23 +42,44 @@
   prTitle: string;
   prDescription: string;
   prompt?: string;
+  isDeleted?: boolean;
 };
 
+// export function assemblesContentToAnalyze(parsedDiff: File[], prDetails: PRDetails): Content[] {
+//   const content = parsedDiff.filter(removeDeletedFilter).reduce((acc, file) => {
+//     acc.push({
+//       filename: file.to ?? '',
+//       content: file.chunks.map(chunkToContent).join('\n'),
+//       prTitle: prDetails.title,
+//       prDescription: prDetails.description ?? ''
+//     });
+//     return acc;
+//   }, [] as Content[]);
+
+//   if (!content?.length) {
+//     core.info('No files found to analyze');
+//     process.exit(0);
+//   }
+
+//   return content;
+// }
 export function assemblesContentToAnalyze(parsedDiff: File[], prDetails: PRDetails): Content[] {
-  const content = parsedDiff.filter(removeDeletedFilter).reduce((acc, file) => {
+  const content = parsedDiff.filter(removeInvalidFilter).reduce((acc, file) => {
+    const isDeleted = !file.to || file.to === '/dev/null';
+    // Inclui tanto arquivos modificados quanto deletados
     acc.push({
-      filename: file.to ?? '',
+      filename: file.to || file.from || '',
```

Comment:
⚠️
Usar 'file.to || file.from' para determinar o 'filename' pode introduzir um comportamento inesperado caso 'file.from' não seja apropriadamente verificado em outros contextos que dependam desta propriedade.
Garantir que a lógica de fallback para propriedades de objeto considera todas as implicações para evitar erros em contextos dependentes.